### PR TITLE
Bump patternfly-react 2.29.17  patternfly-react-extensions 2.16.19

### PR DESCRIPTION
**Pull request by lucasponce**

### CODE REVIEW

**Comment by lucasponce** [comment]
I run a smoke test and no issues so far, but let me know if this works ok for you.

**Comment by aljesusg** [comment]
The requirement for both packages are  "patternfly": "^3.58.0" so it's ok.
I tested the UI and it's all ok.

## Build actual master

- 380M	node_modules
- 33M	build

File sizes after gzip:

-   953.64 KB  build/static/js/main.b003429c.js
-   79.51 KB   build/static/css/main.d96920f5.css

## Build with bump

- 376M	node_modules
- 33M	build

File sizes after gzip:

-  962.73 KB  build/static/js/main.7e21aedf.js
-   80.09 KB   build/static/css/main.d1566447.css

**Comment by aljesusg** [comment]
maybe we should start thinking about change the package.json to --max_old_space_size=4096 ^^

**Comment by mtho11** [comment]
I'm hesitant to merge since travis is having Heap issues:
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

**Comment by hhovsepy** [comment]
Seeing this error in console log of FF and Chrome:
"Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 2019-03-07 16:39:26 +0000 UTC, _f: undefined, _strict: undefined, _locale: [object Object]
"
And after several seconds, about fails, on both browsers:
screenshot from 2019-03-06 17-36-39

**Comment by mtho11** [comment]
nice find @hhovsepy. 
I guess we wait on this upgrade.

**Comment by israel-hdez** [comment]
@hhovsepy Console error, and the about box error are already reported and unrelated to this PR:
https://issues.jboss.org/browse/KIALI-2501
https://issues.jboss.org/browse/KIALI-2502

**Comment by lucasponce** [comment]
Going to see if I can increase the memory in the nvm build.
It seems a simple parameter doesn't work.

**Comment by aljesusg** [comment]
This travis error sounds familiar @lucasponce  https://github.com/kiali/kiali-ui/pull/951#issuecomment-456779324 ^^

**Comment by jshaughn** [comment]
> nice find @hhovsepy.

@mtho11 the Deprecation warning was there prior to this change.

**Comment by hhovsepy** [comment]
> > nice find @hhovsepy.
> 
> @mtho11 the Deprecation warning was there prior to this change.

and the about box problem as well :)

**Comment by hhovsepy** [comment]
Otherwise UI automation regression run did not catch anything else.

**Comment by lucasponce** [comment]
@mtho11 @aljesusg 
Ok, I've sent a last modification in travis to apply the 
```
    - NODE_OPTIONS=--max_old_space_size=4096
```
globaly, and travis worked well 3 times in a row.

As commented, this PR should be a "minor" one just to bump the libraries.
Any other minor flaw I suggest we can address it with a following PR with more context.

**Comment by mtho11** [comment]
@lucasponce I saw that solution yesterday, just didn't have time to try it.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 dist: trusty
 node_js:
-  - 10
+  - '10'
```
Comment (mtho11) [inline .travis.yml]: I didn't think that you had to put quotes around node.js version

**Pull request merged**